tests/sceneManager_test.cpp

```diff
@@ -39,22 +40,73 @@ TEST_F(SceneManagerTest, EnergyCollectorPositionsCheck)
     ASSERT_EQ(manager.getEnergyCollectors().size(), constants::kPopulation) << "wrong numebr of energy collectors";
 
     RayHitData hitData;
-    Ray straightUp(core::Vec3(0, 0, 0), core::Vec3(0, 0, 1));
+    Ray straightUp(kVZero, core::Vec3(0, 0, 1));
 
     ASSERT_TRUE(performHitAtEnergyCollectors(straightUp, &hitData)) << "no hit: " << straightUp;
 
     core::Vec3 referenceCollisionPoint(0, 0, manager.simulatiorRadius() / 2 - manager.collectorRadius());
     ASSERT_EQ(hitData.collisionPoint, referenceCollisionPoint) << "point from: " << straightUp;
 
-    Ray parallelXAxis(core::Vec3(0, 0, 0), core::Vec3(-1, 0, 0));
+    Ray parallelXAxis(kVZero, core::Vec3(-1, 0, 0));
     ASSERT_TRUE(performHitAtEnergyCollectors(parallelXAxis, &hitData)) << "no hit: " << parallelXAxis;
```

Comment:
I still see the `<< "..."` that do not give you any useful information about the error. Please remove. Note that when the error happens you will see `parallelXAxis` name so it should be obvious which test failed. To fix you need to go to the code and verify anyway, so printing what `parallelXAxis` actually is does not make sense.

Reply:
I removed all strings next to ASSERT_TRUE in this file. I will check if this issue exist in other cases